### PR TITLE
Renamed  to DOCKER_OPS to DOCKER_OPTS

### PR DESCRIPTION
**Pull request by eeyun**
Quick naming convention change to remain consistent with external projects.

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/hab/src/command/studio.rs [hunk 1]
```diff
@@ -197,13 +197,13 @@ mod inner {
 
         // All the args already placed in `cmd_args` are things that we don't want to insert again.
         // Later args such as `--env` will overwrite any options (potentially) set mistakenly here.
-        if let Ok(ops) = henv::var(DOCKER_OPS) {
+        if let Ok(ops) = henv::var(DOCKER_OPTS) {
```
Comment (smurawski) [inline components/hab/src/command/studio.rs]: @eeyun since we are in here, why not change the ops to opts for the local variable name?

Comment (Hoverbear) [inline components/hab/src/command/studio.rs]: Agreed!

Comment (eeyun) [inline components/hab/src/command/studio.rs]: Yeah that makes sense. One moment

**Comment by smurawski** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/171476664).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/171476664).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**